### ISSUE TITLE
Wrong implementation of TranAD and OmniAnomaly

### ISSUE THREAD

**Issue by florianebmeier** [thedatumorg/TSB-AD#55]
I found two wrong implementations in the repository. 

The first one is a quick one. The original TranAD paper used MinMaxscaling to have targetvalues between 0 and 1 and then a sigmoid function at the end to restrict to that range. You also took the sigmoid function but I believe you did not apply MinMaxscaling for their model. So you either have to change it by removing the sigmoid or doing MinMaxscaling. 
(In my personal tests removing the sigmoid it still performed rather middle of the pack)

The second one is the OmniAnomaly implementation. And I don't really understand what is going on here but the implementation is just completely wrong. 
It is missing the planar flow. 
The decoder is missing the RNN.
The initialisation of the hidden init is wrong. 
The STD parameter is implemented differently. 
The Anomaly Score is the MSE and not the log likelihood as in the original paper. 
For some reason there is also a sigmoid at the end. 

I do not know specifically where you have this implementation from but it does not look at all like the original OmniAnomaly. I personally did not implement the OmniAnomaly and test it on here, but it might be worth it to properly translate the original OmniAnomaly code from Tensorflow to pytorch and rerun it.

**Comment by qhliu26** [comment]
Update the implementation in this commit https://github.com/thedatumorg/TSB-AD/commit/7e2dfd81302f69558e215ebf67fd45db61e8b679
